docs/interpreter/jdbc.md

```diff
@@ -537,20 +537,40 @@ Here are some examples you can refer to. Including the below connectors, you can
 When Zeppelin server is running with authentication enabled, then the interpreter can utilize Hive's user proxy feature i.e. send extra parameter for creating and running a session ("hive.server2.proxy.user=": "${loggedInUser}"). This is particularly useful when multiple users are sharing a notebooks.
 
 To enable this set following:
+
   - `zeppelin.jdbc.auth.type` as `SIMPLE` or `KERBEROS` (if required) in the interpreter setting.
   - `${prefix}.proxy.user.property` as `hive.server2.proxy.user`
-  Example configuration
 
-  *Properties*
 
-  | name                      | value                                                                                             |
-  |:------------------------- |:--------------------------------------------------------------------------------------------------|
-  | hive.driver               | org.apache.hive.jdbc.HiveDriver                                                                   |
-  | hive.password             |                                                                                                   |
-  | hive.url                  | jdbc:hive2://hive-server-host:2181/;serviceDiscoveryMode=zooKeeper;zooKeeperNamespace=hiveserver2 |
-  | hive.proxy.user.property  | hive.proxy.user.property                                                                          |
-  | zeppelin.jdbc.auth.type   | SIMPLE                                                                                            |
+  Example configuration
 
+##### Properties
```

Comment:
Yeah you need to use static table tag to generate the table. Seems it's working well now. 
![screen shot 2017-05-04 at 11 21 13 am](https://cloud.githubusercontent.com/assets/10060731/25688632/d11082ee-30bb-11e7-8cb7-ac871154db99.png)